apps/entity-service/src/entities/dto/nursery-report.dto.ts

```diff
@@ -0,0 +1,237 @@
+import { NurseryReport } from "@terramatch-microservices/database/entities";
+import { EntityDto, AdditionalProps } from "./entity.dto";
+import { pickApiProperties } from "@terramatch-microservices/common/dto/json-api-attributes";
+import { JsonApiDto } from "@terramatch-microservices/common/decorators/json-api-dto.decorator";
+import { ApiProperty } from "@nestjs/swagger";
+import { MediaDto } from "./media.dto";
+import { User } from "@sentry/nestjs";
+
+@JsonApiDto({ type: "nurseryReports" })
+export class NurseryReportLightDto extends EntityDto {
+  constructor(nurseryReport?: NurseryReport, props?: AdditionalNurseryReportLightProps) {
+    super();
+    if (nurseryReport != null) {
+      this.populate(NurseryReportLightDto, {
+        ...pickApiProperties(nurseryReport, NurseryReportLightDto),
+        lightResource: true,
+        // these two are untyped and marked optional in the base model.
+        createdAt: nurseryReport.createdAt as Date,
+        updatedAt: nurseryReport.createdAt as Date,
+        ...props
+      });
+    }
+  }
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated nursery name"
+  })
+  nurseryName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated nursery uuid"
+  })
+  nurseryUuid: string | null;
+
+  @ApiProperty()
+  frameworkKey: string | null;
+
+  @ApiProperty()
+  frameworkUuid: string | null;
+
+  @ApiProperty()
+  status: string;
+
+  @ApiProperty()
+  updateRequestStatus: string;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project name"
+  })
+  projectName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project uuid"
+  })
+  projectUuid: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation name"
+  })
+  organisationName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation uuid"
+  })
+  organisationUuid: string | null;
+
+  @ApiProperty()
+  updatedAt: Date;
+
+  @ApiProperty({ nullable: true })
+  submittedAt: Date | null;
+
+  @ApiProperty({ nullable: true })
+  taskId: number | null;
```

Comment:
I believe we want to be using `taskUuid` here. See the project report entity / dto.